.github/workflows/icon-review.yml

```diff
@@ -54,19 +54,25 @@ jobs:
           echo image_url=$IMAGE_URL >> $GITHUB_ENV
         shell: bash
 
-      - name: Post review in PR ✍️
-        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+      - name: Find comment if exists 🕵️
+        uses: peter-evans/find-comment@3eae4d37986fb5a8592848f6a574fdf654e61f9e # v3.1.0
+        id: fc
         with:
-          script: |
-            const image_url = "${{ env.image_url }}";
-            console.log('URL: '+image_url);
-            const commentBody = '## Preview \n\nThank you for creating a pull request. This preview shows you how your changes will look on the different themes: \n\n ![Generated Preview](' + image_url + ') \n\n You can find more information how to contribute in the [contribution guidelines](https://github.com/material-extensions/vscode-material-icon-theme/blob/main/CONTRIBUTING.md).';
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: 'github-actions[bot]'
+          body-includes: This preview shows you how your changes will look on the different themes
 
-            github.rest.issues.createComment({
-              issue_number: context.issue.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              body: commentBody
-            })
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Post or update comment in PR ✍️
+        uses: peter-evans/create-or-update-comment@71345be0265236311c031f5c7866368bd1eff043 # v4.0.0
+        with:
+          comment-id: ${{ steps.fc.outputs.comment-id }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            ## Preview
+
+            Thank you for creating a pull request. This preview shows you how your changes will look on the different themes:
+
+            ![Generated Preview](' + image_url + ')
```

Comment:
Thanks for the PR, I think it would be **much better** if the image could be collapsed and expanded

```suggestion
            <details>
              <summary>Generated Preview</summary>

              ![Generated Preview](${{ env.image_url }})
            </details>
```

Reply:
Hmmm, would we want it to be visible by default? I would, but I also feel that sorta defeats the point.

Reply:
That's trickier, but probably possible. I'll take a look.

Reply:
Is this what you were thinking?:
If I save line 42 in `GITHUB_ENV`, then I could do `var > 10 && "" || "open"`? Is that logic right? I feel like I got it backward.

Reply:
Oh, I'll need to fix the img_url part either way. Thanks for catching that, by the way

Reply:
> Is this what you were thinking?:
> If I save line 42 in `GITHUB_ENV`, then I could do `var > 10 && "" || "open"`? Is that logic right? I feel like I got it backward.

```
if icons > 10:
    markdown.append(
        <details>
          <summary>Generated output</summary>

        <Ur img>
        </details>
    )
else
    markdown.apeend(<Ur img>)
```

Reply:
DumbScript™®